### PR TITLE
Am adaugat tema5

### PR DESCRIPTION
**Pull request by curtnazarEliz**

### CODE REVIEW

**Review by UdreaAnd — APPROVED** [review]

**Review by tincageorgianaalina — APPROVED** [review]

**Review by 95teo — CHANGES_REQUESTED** [review]
Let's try to make a call these days. It looks like you've created an entire new maven project and it's not necessary. We only need the java classes. Thanks!:D

File: src/main/java/elizCurtnazar/driver/BrowserManager.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public static ChromeDriver createDriver(){ //cream un obiect de tipu driver. cream pagina de chrome. o inscriem intro clasa
+        WebDriverManager.chromedriver().setup(); // initializam
+        return new ChromeDriver(); // returnam o instanta de chromedriver
+    }
+
+    public static void closeChromeDriver(ChromeDriver driver){
+
+        driver.quit(); //inchide driverul
+    }
+
```
Comment (95teo) [inline src/main/java/elizCurtnazar/driver/BrowserManager.java:16]: No need to add this two blank spaces here. Please remove them. Thanks!

File: src/test/java/elizCurtnazar/tests/DemoQaManager.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        }
+        System.out.println(list);
+        WebElement homeLink = driver.findElement(By.id("simpleLink"));
+        homeLink.click();
+        return homeLink;
+
+
+    }
+}
+//       WebElement webTable = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[4]"));
```
Comment (95teo) [inline src/test/java/elizCurtnazar/tests/DemoQaManager.java:149]: Why are these lines commented? Aren't they used anymore? If so, please remove them. Thanks!

File: src/test/java/elizCurtnazar/tests/DemoQaManager.java [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        System.out.println(question.getText());
+        WebElement yesAnswer = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[2]/label"));
+        yesAnswer.click();
+        WebElement answer = driver.findElement(By.cssSelector(".mt-3"));
+        answer.getText();
+        System.out.println(answer.getText());
+    }
+
+
+    public void WebTablesLink(String link) {
```
Comment (95teo) [inline src/test/java/elizCurtnazar/tests/DemoQaManager.java:85]: Please use camelCase when naming methods as per Java conventions. Check it for every method that you wrote. Thanks!

File: src/test/java/elizCurtnazar/tests/BothPagesTests.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+        demoqaPage.selectTextBox();
+        demoqaPage.selectCheckBox();
+        demoqaPage.selectRadioButton();
+        demoqaPage.WebTablesLink("webtables");
+        demoqaPage.ButtonsLink("buttons");
+        demoqaPage.LinkElementLink("links");
+
+
+//        herokuapp.goToTheFirstPage();
```
Comment (95teo) [inline src/test/java/elizCurtnazar/tests/BothPagesTests.java:21]: Why are these lines commented? They shouldn't be in order to run the methods for heroku.

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/test/java/elizCurtnazar/tests/DemoQaManager.java (hunk 2 above)
Comment (curtnazarEliz) [reply src/test/java/elizCurtnazar/tests/DemoQaManager.java:149]: Sunt unele varianta pe care vroiam sa le fac. Pot sa le las sau trebuie neaparat sterse?

File: src/main/java/elizCurtnazar/driver/BrowserManager.java (hunk 1 above)
Comment (curtnazarEliz) [reply src/main/java/elizCurtnazar/driver/BrowserManager.java:16]: done.

File: src/test/java/elizCurtnazar/tests/BothPagesTests.java (hunk 4 above)
Comment (curtnazarEliz) [reply src/test/java/elizCurtnazar/tests/BothPagesTests.java:21]: done

File: src/test/java/elizCurtnazar/tests/BothPagesTests.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package elizCurtnazar.tests;
+
+import elizCurtnazar.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BothPagesTests {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createDriver();
+        TestPagesHerokuappManager herokuapp = new TestPagesHerokuappManager("https://testpages.herokuapp.com/styled/index.html", driver);
```
Comment (95teo) [inline src/test/java/elizCurtnazar/tests/BothPagesTests.java:9]: Move this line after demoqa methods and before the heroku methods in order to have functional run. Thanks!

File: src/test/java/elizCurtnazar/tests/DemoQaManager.java (hunk 2 above)
Comment (95teo) [reply src/test/java/elizCurtnazar/tests/DemoQaManager.java:152]: In mod normal, s-ar sterge. Insa ai putea sa lasi un comentariu in care sa spui ca sunt alte variante pe care le-ai incercat sau ceva de genul si ca le pastrezi pentru istoric.

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/test/java/elizCurtnazar/tests/BothPagesTests.java (hunk 5 above)
Comment (curtnazarEliz) [reply src/test/java/elizCurtnazar/tests/BothPagesTests.java]: Done.

**Review by sorinfoca — APPROVED** [review]

**Review by 95teo — APPROVED** [review]

**Pull request merged**